Dockerfile

```diff
@@ -30,10 +30,11 @@ RUN cd /opt && tar -xvzf android-sdk.tgz
 RUN cd /opt && rm -f android-sdk.tgz
 
 # Grab what's needed in the SDK
+RUN echo y | android update sdk --no-ui --all --filter tools > /dev/null 
```

Comment:
Right.

The story is a bit weird.

Running `RUN echo y | android update sdk --no-ui --all --filter tools` produces following output.
```
...
Installing Archives:
  Preparing to install archives
  Skipping 'Android SDK Tools, revision 25.1.7'; it depends on 'Android SDK Platform-tools, revision 24' which was not installed.
  Done. Nothing was installed.
```

This says that install was skipped and nothing was installed. But, if I remove this line, the lint check does not pass.

@emanuelez Do you know why this happens?